tools/third_party/pytest/src/_pytest/_version.py

```diff
@@ -2,3 +2,4 @@
 # file generated by setuptools_scm
 # don't change, don't track in version control
 version = '7.0.1'
```

Comment:
It seems a bit surprising we're adding a file that specifically says it shouldn't be checked in.